lib/active_merchant/billing/gateways/omise.rb

```diff
@@ -0,0 +1,298 @@
+require 'active_merchant/billing/rails'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class OmiseGateway < Gateway
+      API_VERSION = '1.0'
+      API_URL     = 'https://api.omise.co/'
+      VAULT_URL   = 'https://vault.omise.co/'
+
+      STANDARD_ERROR_CODE_MAPPING = {
```

Comment:
Are there any other standard errors that come back? I noticed that the only error being changed is this one - do the other errors all map to ActiveMerchant's standard error codes?